sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_directory_client.hpp

```diff
@@ -62,23 +62,23 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
      * @param directoryUri The URI of the file system this client's request targets.
      * @param options Optional parameters used to initialize the client.
      */
-    explicit DirectoryClient(
+    explicit DataLakeDirectoryClient(
         const std::string& directoryUri,
         const DataLakeClientOptions& options = DataLakeClientOptions());
 
     /**
-     * @brief Create a FileClient from current DirectoryClient
+     * @brief Create a FileClient from current DataLakeDirectoryClient
      * @param path Path of the file under the directory.
      * @return FileClient
      */
-    FileClient GetFileClient(const std::string& path) const;
+    DataLakeFileClient GetDataLakeFileClient(const std::string& path) const;
```

Comment:
```suggestion
    DataLakeFileClient GetFileClient(const std::string& path) const;
```

Reply:
It's `GetShareDirectoryClient` in file share.

Reply:
And who wrote that in file share?

Reply:
https://github.com/Azure/azure-sdk-for-cpp/pull/1033, PR approved.

Reply:
I missed that. I take the approval back.